src/main/java/org/broadinstitute/hellbender/tools/sv/LocusDepth.java

```diff
@@ -79,6 +76,12 @@ public char getRefCall() {
     public int getGDepth() { return depths[2]; }
     public int getTDepth() { return depths[3]; }
 
+    @Override
+    public LocusDepth extractSamples( final List<String> sampleNames,
+                                      final Object header ) {
+        return this;
```

Comment:
I didn't realize before that this class didn't store the sample name. I think it should (just like BAF/SR/PE) and return null if the sample is not in the given list.

Reply:
No we haven't been using this data type yet, so you are free to modify it